### PR TITLE
[ZTP] Add DHCP_L2 and DHCPV6_L2 traps registration

### PR DESCRIPTION
**Pull request by liorghub**
**What I did**
Add DHCP_L2 and DHCPV6_L2 traps registration.

**Why I did it**
In order to enable DHCP packets to CPU on L2 interfaces for getting IP address and DHCP ZTP options.

**How I verified it**
DHCP packets arrive to CPU and inband interface is getting IP address. In addition DHCP ZTP options are received and handled.
I also verified that traps are registered when ZTP operation starts and being deleted when ZTP finishes its operation.

### CODE REVIEW

File: src/usr/lib/ztp/dhcp/inband-ztp-ip [hunk 1]
```diff
@@ -18,6 +18,8 @@
 # Purpose:      Used by dhclient-script to configure IP address for front panel interface
 #
 
+export PATH=/usr/local/bin:$PATH
```
Comment (dgsudharsan) [inline src/usr/lib/ztp/dhcp/inband-ztp-ip:21]: Can you please explain why this change is needed?

Comment (liorghub) [reply src/usr/lib/ztp/dhcp/inband-ztp-ip:21]: When this DHCP hook is being called, it executes CLI commands. CLI executable is under /usr/local/bin and it must be added to PATH in order for the hook to have access to the CLI executable. I consulted code owner from Broadcom about this issue and this solution was suggested.

**Review by dgsudharsan — APPROVED** [review]

File: src/usr/lib/ztp/dhcp/inband-ztp-ip (hunk 1 above)
Comment (dgsudharsan) [reply src/usr/lib/ztp/dhcp/inband-ztp-ip:21]: Understood.
